tests/newsModel.test.js

```diff
@@ -0,0 +1,23 @@
+const NewsModel = require("../src/newsModel")
+
+describe("NewsModel", ()=>{
+	let model;
+	beforeEach(()=>{
+		model = new NewsModel();
+	})
+
+	it("creates an instance of itself", ()=>{
+		expect(model).toBeInstanceOf(NewsModel)
+	})
+
+	it("has a getNews() method", ()=>{
```

Comment:
maybe this test could be written with more detail. It is actually testing that the getNews() method returns an empty to array to begin with. Is it actually testing whether the method exists?